src/panels/config/automation/structs.ts

```diff
@@ -0,0 +1,9 @@
+import { object, optional, number } from "superstruct";
+
+export const forDictStruct = object({
+  days: optional(number()),
+  hours: optional(number()),
+  minutes: optional(number()),
+  seconds: optional(number()),
+  milliseconds: optional(number()),
```

Comment:
We only support hours, minutes and seconds.

![image](https://user-images.githubusercontent.com/1444314/144903675-82e19947-92f9-4a04-a80a-4aac219056f9.png)